docs/blog/2019-10-17-how-gatsby-can-power-your-site-offline-even-without-javascript/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "How Gatsby can power your site offline - even without JavaScript"
```

Comment:
```suggestion
title: "How Gatsby Can Power your Site Offline - Even without JavaScript"
```
We use title case in the Style Guide, what do you think about using it here? It's not super consistent in the blog.

Reply:
I'm okay either way! https://titlecase.com says all the words should be uppercase here though?

Reply:
I've seen it go both ways. We can leave it for the blog though, no worries!